File: src/main/java/pe/com/test_qsystem/distribution_center/impl/LurinDistributionCenterImpl.java
package pe.com.test_qsystem.distribution_center.impl;

import pe.com.test_qsystem.distribution_center.DistributionCenter;

public class LurinDistributionCenterImpl implements DistributionCenter {

    @Override
    public boolean support(String center) {
        return center.equalsIgnoreCase("LURIN");
    }

    @Override
    public void operate(String center) {
        System.out.println("Operate in Lurin distribution center.");
    }
    
}

File: src/main/kotlin/pe/com/qsystem/test_qsystem/distribution_center/impl/LurinDistributionCenterImpl.kt
package pe.com.qsystem.test_qsystem.distribution_center.impl

import org.slf4j.LoggerFactory
import pe.com.qsystem.test_qsystem.distribution_center.DistributionCenter

class LurinDistributionCenterImpl : DistributionCenter {
        
    private val log = LoggerFactory.getLogger(javaClass)

    override fun support(center: String) = center.contentEquals("LURIN", true)

    override fun operate(center: String) {
        log.info("Operate in Lurin distribution center.")
    }
}